plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMaterializedViewProperties.java

```diff
@@ -21,35 +21,45 @@
 import java.util.Map;
 import java.util.Optional;
 
-import static io.trino.spi.session.PropertyMetadata.stringProperty;
+import static io.trino.spi.session.PropertyMetadata.booleanProperty;
 
 public class IcebergMaterializedViewProperties
 {
-    public static final String STORAGE_SCHEMA = "storage_schema";
+    public static final String SEPARATE_STORAGE_TABLE = "separate_storage_table";
 
     private final List<PropertyMetadata<?>> materializedViewProperties;
 
     @Inject
-    public IcebergMaterializedViewProperties(IcebergConfig icebergConfig, IcebergTableProperties tableProperties)
+    public IcebergMaterializedViewProperties(
+            @EnableMaterializedViewSeparateStorageTable boolean enableMaterializedViewSeparateStorageTable, // for tests of legacy materialized views
```

Comment:
Don’t add this. We don’t want special code just for testing. We’re removing this pattern from the code base.

Reply:
Agreed this shouldn't be here, and should eventually move to test code. I didn't see the urgency to do this within one PR. What do you think?

Reply:
I don’t think we should add more test-only annotations to production code. This is an anti-pattern and we don’t want people to see this and think it is ok.

We’re keeping legacy code to create these views, so it’s not clear how much this change is helping at this stage.

Reply:
I think I could save you some time on your next review round, if you gave me a hint how you prefer want to see this.

Reply:
Can we create the legacy view manually in the test code? Or maybe we start an old version of Trino in Testcontainers to write the legacy view?